### PR TITLE
New release announcement

### PR DESCRIPTION
**Pull request by dhh**

### CODE REVIEW

File: _posts/2018-04-09-Rails-5-2-0-final.markdown [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+You can read in even more detail about everything that's [new in Rails 5.2 in the newly finished release notes](http://edgeguides.rubyonrails.org/5_2_release_notes.html).
+
+Note that [rails/master](https://github.com/rails/rails) development is now targeting Rails 6.0.
+
+Many thanks to Rails core, Rails contributors, and everyone else who've helped with code, documentation, bug reports, and whatever else to get Rails 5.2.0 out the door. It's amazing to have [over 500 code contributors](http://contributors.rubyonrails.org/edge/contributors) with fingerprints on this release.
+
+## Feature highlights
+
+It’s been too hard to deal with file uploads in Rails for too long. Sure, there’s been a lot of fine plugins available,
+but it was overdue that we incorporated something right into the framework. So now we have!
```
Comment (eileencodes) [inline _posts/2018-04-09-Rails-5-2-0-final.markdown:22]: > So now we have!

I think this sentence seems incomplete?

Comment (dhh) [reply _posts/2018-04-09-Rails-5-2-0-final.markdown:22]: That's a sentence connected to "it was overdue that we incorporated something". "So now we have [incorporated something right into the framework]".

File: _posts/2018-04-09-Rails-5-2-0-final.markdown [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+This release comes just in time for [RailsConf](https://railsconf.com/), which features sessions on the new [encrypted credentials](https://railsconf.com/program/sessions#session-627), [a code review of Active Storage](https://railsconf.com/program/sessions#session-562), [advice on how to upgrade to a new Rails version](https://railsconf.com/program/sessions#session-588), and a [lot](https://railsconf.com/program/sessions#session-552) [of](https://railsconf.com/program/sessions#session-568) [Webpack](https://railsconf.com/program/sessions#session-549) talks.
+
+You can read in even more detail about everything that's [new in Rails 5.2 in the newly finished release notes](http://edgeguides.rubyonrails.org/5_2_release_notes.html).
+
+Note that [rails/master](https://github.com/rails/rails) development is now targeting Rails 6.0.
+
+Many thanks to Rails core, Rails contributors, and everyone else who've helped with code, documentation, bug reports, and whatever else to get Rails 5.2.0 out the door. It's amazing to have [over 500 code contributors](http://contributors.rubyonrails.org/edge/contributors) with fingerprints on this release.
```
Comment (rafaelfranca) [inline _posts/2018-04-09-Rails-5-2-0-final.markdown]: The link to the edge contributors now points to 6.0. I'll fix it as soon the 5.2.0 link is generated.

**Comment by dhh** [comment]
Ah! Should make sure that the 500 number is accurate after that link is
available :D

**Comment by rafaelfranca** [comment]
The new link for the over 400 contributors is there.

**Pull request merged**